### ISSUE TITLE
Field does not show correct activation date

### ISSUE THREAD

**Issue by animaux** [symphonycms/members#306]
When a user is activated the activation date/time is shown in the field. This does not seem to reflect the actual activation date but is set to whenever the entry its is was last saved.

**Comment by nitriques** [comment]
So the activation date display the last modified date ?

**Comment by nitriques** [comment]
Seems like it !!

**Comment by nitriques** [comment]
The timestamp is updated when the activation field is saved. It looks like a feature, not a bug.

What about changing the label by "Last activated" ?

**Comment by animaux** [comment]
Hmm. Does not make sense to me. Why should the activation be renewed every time the entry is saved? That would simply mimic the function of the system saved date. 

It makes a lot more sense to me to only store/save the date of the initial activation, especially since there is no way to deactivate a user again if I get it right.

**Comment by nitriques** [comment]
Yeah but looking at the code, it is clear to me that it is how it was designed. Now, do we still need this ? We would need to git blame the thing before taking a conscious decision.

**Comment by animaux** [comment]
The blame game!

I’ll have a look on monday. Not feeling conscious enough anymore today. Thanks for your insights! Good night `:)`

**Comment by nitriques** [comment]
nite nite !

**Comment by animaux** [comment]
Only @brendo and @cz have worked on the activation field file. Maybe they remember if this was a conscious decision or can tell if it is a bug?

**Comment by cz** [comment]
Looks like the timestamp came along after I worked on this field, so I can't speak to the original intent. If I had to guess, though, it sounds like a bug.

**Comment by animaux** [comment]
Thanks for chiming i Craig, I do feel the same.
